packages/yoga/src/AutoComplete/web/AutoComplete.jsx

```diff
@@ -200,33 +229,51 @@ const AutoComplete = React.forwardRef(
               second.toLowerCase().indexOf(inputValue)
                 ? -1
                 : 1,
-            )
-            .slice(0, 6);
+            );
 
-          const hasSuggestion = isOpen && Boolean(suggestionList.length);
+          if (!!inputValue && isOpen) {
```

Comment:
As you are checking the truthy value, why not using like this?
```suggestion
          if (inputValue && isOpen) {
```

Reply:
Hey, @diegomarcuz.
This is just to be sure that the `inputValue` will always be a boolean, to prevent unexpected JavaScript behavior. This is the same as `Boolean(inputValue)`.

Reply:
I understood, my comment is just a matter of semantic, but it is fine.